Project2015To2017.Core/Reading/ProjectPropertiesReader.cs

```diff
@@ -1,33 +1,42 @@
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Xml.Linq;
 using Microsoft.Extensions.Logging;
 using Project2015To2017.Definition;
-using Project2015To2017.Transforms;
 
 namespace Project2015To2017.Reading
 {
 	public sealed class ProjectPropertiesReader
 	{
-		private readonly ILogger _logger;
-
 		private static readonly string[] RemoveMSBuildImports =
 		{
 			@"$(MSBuildToolsPath)\Microsoft.CSharp.targets",
 			@"$(MSBuildBinPath)\Microsoft.CSharp.targets",
 			@"$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props",
 		};
+		private static readonly Guid TestProjectTypeGuid = Guid.ParseExact("3AC096D0-A1C2-E12C-1390-A8335801FDAB", "D");
```

Comment:
Shouldn't we move this to the same location as `IsWindowsPresentationFoundationProject()`? Just to keep all of the project type guids in 1 place?

Reply:
There is no point in it being accessible from any other place other than its only usage.